templates/typescript-app/package.json

```diff
@@ -23,10 +23,10 @@
   "devDependencies": {
     "cdk8s-cli": "{{ cdk8s_cli_spec }}",
     "@types/node": "^14",
-    "@types/jest": "^26",
```

Comment:
The latest `yaml` version [added](https://github.com/eemeli/yaml/commit/374c19cfdc051ca81c5a1ce12ca733c2e2a9ec93) require of `node:process`, which isn't supported in jest 26.